website/i18n/en.json

```diff
@@ -457,1339 +473,1339 @@
         "title": "Fixture Groups",
         "sidebar_label": "Fixture Groups"
       },
-      "version-12.0-controlling-fixtures/using-the-select-buttons-and-wheels": {
+      "version-12.0/controlling-fixtures/version-12.0-using-the-select-buttons-and-wheels": {
         "title": "Using the Select Buttons and Wheels",
         "sidebar_label": "Using the Select Buttons and Wheels"
       },
-      "version-12.0-controlling-fixtures/viewing-and-editing-fixture-values": {
+      "version-12.0/controlling-fixtures/version-12.0-viewing-and-editing-fixture-values": {
         "title": "Viewing and Editing Fixture Values",
         "sidebar_label": "Viewing and Editing Fixture Values"
       },
-      "version-12.0-cue-lists": {
+      "version-12.0/version-12.0-cue-lists": {
         "title": "Cue Lists",
         "sidebar_label": "Cue Lists"
       },
-      "version-12.0-cue-lists/copying-moving-linking-and-deleting": {
+      "version-12.0/cue-lists/version-12.0-copying-moving-linking-and-deleting": {
         "title": "Copying, Moving, Linking and Deleting",
         "sidebar_label": "Copying, Moving, Linking and Deleting"
       },
-      "version-12.0-cue-lists/creating-a-cue-list": {
+      "version-12.0/cue-lists/version-12.0-creating-a-cue-list": {
         "title": "Creating a Cue List",
         "sidebar_label": "Creating a Cue List"
       },
-      "version-12.0-cue-lists/cue-list-options": {
+      "version-12.0/cue-lists/version-12.0-cue-list-options": {
         "title": "Cue List Options",
         "sidebar_label": "Cue List Options"
       },
-      "version-12.0-cue-lists/cue-list-playback": {
+      "version-12.0/cue-lists/version-12.0-cue-list-playback": {
         "title": "Cue List Playback",
         "sidebar_label": "Cue List Playback"
       },
-      "version-12.0-cue-lists/cue-list-timing": {
+      "version-12.0/cue-lists/version-12.0-cue-list-timing": {
         "title": "Cue List Timing",
         "sidebar_label": "Cue List Timing"
       },
-      "version-12.0-cue-lists/editing-cue-lists": {
+      "version-12.0/cue-lists/version-12.0-editing-cue-lists": {
         "title": "Editing Cue Lists",
         "sidebar_label": "Editing Cue Lists"
       },
-      "version-12.0-cue-lists/theatre-programming": {
+      "version-12.0/cue-lists/version-12.0-theatre-programming": {
         "title": "Theatre Programming Quick Guide",
         "sidebar_label": "Theatre Programming"
       },
-      "version-12.0-cues": {
+      "version-12.0/version-12.0-cues": {
         "title": "Cues",
         "sidebar_label": "Cues"
       },
-      "version-12.0-cues/copying-moving-linking-and-deleting": {
+      "version-12.0/cues/version-12.0-copying-moving-linking-and-deleting": {
         "title": "Copying, Moving, Linking and Deleting a Cue",
         "sidebar_label": "Copying, Moving, Linking and Deleting"
       },
-      "version-12.0-cues/creating-a-cue": {
+      "version-12.0/cues/version-12.0-creating-a-cue": {
```

Comment:
I know this file is auto generated but annoying it is generated wrong. The way it was previously is correct for the translations to work. We do need the new additions for timeline though. I think this file will have to be fixed manually rather than using the generated one. Happy to do that if that doesn't make sense